### ISSUE TITLE
Bundle version.cmake in release tarballs

### ISSUE THREAD

**Issue by jpo234** [rnpgp/rnp#1377]
## Description

The cmake build is broken, because rnp-0.14.0/cmake/version.cmake is missing.

## Steps to Reproduce

1. Download and unpack rnp-0.14.0.tar.gz
2. run cmake

## Expected Behavior

Build environment is setup

## Actual Behavior

```
cmake ..
CMake Error at CMakeLists.txt:43 (determine_version):
  Unknown CMake command "determine_version".
```

**Comment by dewyatt** [comment]
There's a few things here:
1) 0.14.0 has not been officially released yet and shouldn't have been tagged.
2) Delete the build directory and try again, I just tested and had no issues. Please post the full log if issues persist.

**Comment by jpo234** [comment]
The problem is commit [2bbc59d5b254e9865557dfc758cf80264de2db68](https://github.com/rnpgp/rnp/commit/7f192753ebd85c7a6a1d74ab6e54bc214afba418#diff-251d177e3e7db921c05d14a06c2495601e4bb0e8ccfff429ac77fa6cd65786ec)

That one removed version.cmake and tries to do a download from an external URL. My development system is air gapped and couldn't do the download. I suggest that the file should be included in a release tar ball.

**Comment by dewyatt** [comment]
Glad you got it working! Yes I pulled version.cmake out in that commit but it definitely needs to be bundled into release tarballs, which we have not done yet

**Comment by ni4** [comment]
Fixed via #1443, so closing this.
